homeassistant/components/switcher_kis/strings.json

```diff
@@ -3,6 +3,13 @@
     "step": {
       "confirm": {
         "description": "[%key:common::config_flow::description::confirm_setup%]"
+      },
+      "reauth_confirm": {
+        "description": "Found a Switcher device that requires a token\nEnter your username(Email Address) and token here\nFor security reasons, some devices require a token to operate,\nthe token can be acquired from this link: https://switcher.co.il/GetKey/\nExample of Token: zvVvd7JxtN7CgvkD1Psujw==",
```

Comment:
Please refer to a new section in the docs instead of the URL and example

Reply:
@thecode 
I don't understand what doc is you talking about?

Reply:
@thecode 
You mean I need to make a PR [here ](https://github.com/home-assistant/home-assistant.io/blob/9374fd5bc2fbcab87431bf5327135257133c0f99/source/_integrations/switcher_kis.markdown?plain=1)?
How is it working there? Should It be merged there and here at the same time?

Reply:
Yes, don't worry, they will be merged together

Reply:
Fix the strings.. please see now